### PR TITLE
Finally fixed that AI node to meta

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR At last adress an AI node error in meta, where all the AI bots would get stuck in front off the Telecomms.

## Why It's Good For The Game

It finally address this insaneley annoying stuff on meta

dreamseeker_CUvKGVvKTT

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>
The medibot is finally free to leave the node hell that it used to be.

dreamseeker_RXQLRkYXo9

</details>

## Changelog
:cl:
fix:[Metastation] finally adressed the bug related to bots getting stuck in front off telecomms
/:cl:

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -48199,7 +48192,7 @@
 /mob/living/simple_animal/bot/medbot{
 	auto_patrol = 1;
 	desc = "A little medical robot, officially part of the Nanotrasen medical inspectorate. He looks somewhat underwhelmed.";
-	name = "Inspector Johnson"
+	name = "Sir debugger"
```
Comment (itsmeow) [inline _maps/map_files/MetaStation/MetaStation.dmm:48195]: What

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -68987,6 +68966,18 @@
...
 /area/security/brig)
+"uhq" = (
+/obj/structure/cable/yellow{
+	icon_state = "1-2"
+	},
+/obj/structure/disposalpipe/segment,
+/mob/living/simple_animal/bot/medbot{
+	auto_patrol = 1;
+	desc = "A little medical robot, officially part of the Nanotrasen medical inspectorate. He looks somewhat underwhelmed.";
+	name = "Sir debugger"
```
Comment (itsmeow) [inline _maps/map_files/MetaStation/MetaStation.dmm]: What the fuck is this

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
oh lol

**Comment by itsmeow** [comment]
Mistake
image

**Review by PowerfulBacon — APPROVED** [review]

**Review by Penwin0 — APPROVED** [review]

**Review by itsmeow — APPROVED** [review]

**Pull request merged**